test/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "module": "CommonJS",
+    "target": "ES2018",
```

Comment:
Let's inherit from the root `tsconfig` to avoid drift between properties like `target` (`"extends": "../tsconfig.json"`). We should pick one language level across the whole project, and define it in a single place